File: src/main/java/com/LilG/DataClasses/Meme.java
package com.LilG.DataClasses;

import com.fasterxml.jackson.annotation.JsonCreator;
import com.fasterxml.jackson.annotation.JsonProperty;
import org.jetbrains.annotations.NotNull;

/**
 * The thing that makes the world go round
 * Better than left shark
 */

public class Meme {
	private final String creator;
	private String Meme;

	@JsonCreator
	public Meme(@JsonProperty("creator") String creator,
	            @JsonProperty("Meme") String Meme) {
		this.creator = creator;
		this.Meme = Meme;
	}

	public String getCreator() {
		return creator;
	}

	public String getMeme() {
		return Meme;
	}

	public void setMeme(String meme) {
		Meme = meme;
	}

	@NotNull
	public String toString() {
		return "Creator: " + creator + " - " + Meme;
	}
}

File: FozruciCS/DataStructs/Meme.cs
namespace FozruciCS.DataStructs{
	public struct Meme{
		private readonly string creator;
		private string meme;

		public Meme(string creator, string Meme) {
			this.creator = creator;
			meme = Meme;
		}

		public string getCreator() {
			return creator;
		}

		public string getMeme() {
			return meme;
		}

		public void setMeme(string meme) {
			this.meme = meme;
		}

		public string toString() {
			return "Creator: " + creator + " - " + meme;
		}
	}
}